app/views/subscription/add.phtml

```diff
@@ -218,4 +218,35 @@
 			</div>
 		</div>
 	</form>
+
+	<h2>
+		<?= _t('sub.title.add_dynamic_opml') ?>
+		<span title="<?= _t('sub.category.dynamic_opml') ?>">🗲</span>
+	</h2>
+	<form action="<?= _url('category', 'create') ?>" method="post">
+		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
+		<div class="form-group">
+			<label class="group-name" for="new-category"><?= _t('sub.category') ?></label>
+			<div class="group-controls">
+				<input id="new-category" name="new-category" type="text" autocomplete="off"/>
```

Comment:
`required="required"` missed (fetched by the backend)

Reply:
Fixed